### ISSUE TITLE
News in chronological order?

### ISSUE THREAD

**Issue by hucker75** [FoldingAtHome/fah-web-client-bastet#140]
Could we have the news tab in chronological order (latest at the top)?  Or is it already in a logical order I'm not seeing?

**Comment by jcoffland** [comment]
It's already sorted but somewhat randomly with newer articles having a higher priority.  The idea is that you don't always see only the newest article but the newest are near the top.

**Comment by jcoffland** [comment]
There are plenty of papers being written based on research using F@H but few researches have the time or inclination to write blog posts about them.  I've been pushing for more blog posts but the researchers are all very busy and focused on their projects.  They are happy when F@H is running well and producing results but it's difficult to get scientists motivated to write about their work in layman's terms.  I'll send an internal email now.
